chef/cookbooks/ntp/recipes/default.rb

```diff
@@ -91,7 +91,7 @@
   service "ntp" do
     service_name "ntpd" if node[:platform_family] == "rhel"
     service_name "ntpd" if node[:platform] == "opensuse" || (node[:platform] == "suse" && node[:platform_version].to_f >= 12.0)
-    supports restart: true, status: true, reload: true
+    supports restart: true, status: true, reload: false
```

Comment:
Not that it's wrong, but I fail to understand how this is causing issues as we don't seem to trigger a reload anywhere?

Reply:
```restart``` defaults to false, so this would be an undesired change (running restart is fine, just reload is the problem)